microservices/site-launch/shared/types.ts

```diff
@@ -5,12 +5,15 @@ export enum SiteLaunchLambdaType {
 }
 
 export enum SiteLaunchLambdaStatus {
-  SUCCESS = "success",
-  FAILURE = "failure",
-  PENDING = "pending",
+  SUCCESS_SITE_LIVE = "success - site live",
+  SUCCESS_PROPAGATING = "success - propagating",
+  FAILURE_WRONG_CLOUDFRONT_DISTRIBUTION = "failure - wrong cloudfront distribution",
+  FAILURE_CLOUDFRONT_ALIAS_CLASH = "failure - cloudfront alias clash",
+  PENDING_DURING_SITE_LAUNCH = "pending - during site launch",
+  PENDING_PRE_SITE_LAUNCH = "pending - pre site launch",
```

Comment:
i suddenly realised we don't tag the state here but have to parse the string manually... 
eg: 
```ts 
export type SiteLaunchStatus = { 
  state: "success" | "failure" | "pending"
  message: keyof typeof SiteLaunchLambdaStatus
}  
```

makes the enum more complicated to work with because the state is now implicitly tied to the portion before `-`, which might not be upheld/noticed

Reply:
oo this is a good suggestion, thanks!